src/native/dnmd/readme.md

```diff
@@ -0,0 +1,61 @@
+# .NET MetaData &ndash; DNMD
+
+DNMD represents a suites of tools for manipulating [ECMA-335][ecma_335] defined metadata. It designed to be written in unmanaged code (that is, C/C++) in a modern style. This doesn't mean it is intended to rely on the latest features or libraries. Rather it is written to use canonical C and C++ in a manner that is clear.
+
+DNMD provides the following tools:
+
+- `dnmd` - A static library with no external dependencies that represents the lowest level of reading ECMA-335.
+- `dnmd_interfaces` - A shared library (`.dll`|`.dylib`|`.so`) that consumes `dnmd` and provides higher level .NET APIs. At present the following interfaces are provided:
+  - [`IMetaDataDispenser`][api_dispenser]
+  - [`IMetaDataImport`][api_import] / [`IMetaDataImport2`][api_import2]
+  - [`IMetaDataAssemblyImport`][api_assemblyimport]
+- `dnmd_interfaces_static` - A static library version of `dnmd_interfaces`.
```

Comment:
Do we need its shared variant or can we just used the static lib (or minipal type of object) linked into the projects requiring the functionality?

Reply:
The shared lib is useful for sharing out to other orgs/products that need to parse ECMA metadata.

Also for a debug flow for the cdacreader with DNNE, it would be helpful to have the shared lib.